fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -0,0 +1 @@
+tells you the weather where you are NOT
```

Comment:
Would it be okay if I changed this to the short description I have in the GH repo and the Play Store? ("The weather app that tells you the weather where you aren't")
or is there a reason you changed it?

Reply:
Well, it's all yours – I hear and obey :smile: I don't remember, TBH. And of course you are free to change it anytime. I've just pushed the change.

Reply:
thank you!